### PR TITLE
Change title space encoding to full url encoding for github issues

### LINKED ISSUE #292: Semicolons in titles truncates them
### Describe the Bug

If a show or movie contains a semicolon, for example "Steins;Gate", then trying to add an theme request will truncate the issues title resulting in "[TV SHOW]: Steins"
I've created a patch that fixes this that I'll PR shortly.

### Expected Behavior

The full issue title should be preserved, for the given example it should be "[TV SHOW]: Steins;Gate (2011)" and not "[TV SHOW]: Steins"

### Additional Context

This can be seen in the edited title of this theme request:
LizardByte/ThemerrDB#3849

### PR DESCRIPTION
**Pull request by 1xFF**
## Description

When Themerr creates a github issue template URL, it only encodes spaces through .replace() and not reserved characters.
This PR simply changes .replace() with Uri.EscapeDataString

### Issues Fixed or Closed

Closes #292 

## Type of Change
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency update (updates to dependencies)
- [ ] Documentation update (changes to documentation)
- [ ] Repository update (changes to repository files, e.g. `.github/...`)

## Checklist
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated the in code docstring/documentation-blocks for new or existing methods/components

## Branch Updates
LizardByte requires that branches be up-to-date before merging. This means that after any PR is merged, this branch
must be updated before it can be merged. You must also
[Allow edits from maintainers](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [X] I want maintainers to keep my branch updated

### CODE REVIEW

File: Jellyfin.Plugin.Themerr/ThemerrManager.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-using System;
+﻿using System;
```
Comment (ReenigneArcher) [inline Jellyfin.Plugin.Themerr/ThemerrManager.cs:1]: What has happened here? The BOM changed on the file?

**Comment by ReenigneArcher** [comment]
Thanks for the PR! I can run the workflow once you sign the CLA.

File: Jellyfin.Plugin.Themerr/ThemerrManager.cs (hunk 1 above)
Comment (1xFF) [reply Jellyfin.Plugin.Themerr/ThemerrManager.cs:1]: Huh, never run into that before but yeah, it looks like VS removed the BOM. do you want me to change that?

Comment (ReenigneArcher) [reply Jellyfin.Plugin.Themerr/ThemerrManager.cs:1]: Probably best to revert that for now.

**Comment by 1xFF** [comment]
> Thanks for the PR! I can run the workflow once you sign the CLA.

Done!

**Comment by ReenigneArcher** [comment]
Perhaps we should add a test case here to ensure this doesn't break in the future? https://github.com/LizardByte/Themerr-jellyfin/blob/master/Jellyfin.Plugin.Themerr.Tests/FixtureJellyfinServer.cs#L35

This will also need to be updated. https://github.com/LizardByte/Themerr-jellyfin/blob/982b2a7cf0d9ff76b3140cba03399043320c1745/Jellyfin.Plugin.Themerr.Tests/TestThemerrManager.cs#L486

**Comment by 1xFF** [comment]
I'd love to hard code Steins;Gate into Themerr :)
Can you accept LizardByte/ThemerrDB#3709 so the tests can run?

**Comment by 1xFF** [comment]
Oh no it changed the BOM again...

**Comment by ReenigneArcher** [comment]
I forgot that the fixtures require these items to exist in the database. Which one of these is the correct one?

https://github.com/LizardByte/ThemerrDB/issues?q=is%3Aissue+is%3Aopen+steins%3Bgate+%282011%29

**Comment by 1xFF** [comment]
> > Oh no it changed the BOM again... it doesn't show that in VS's git diff so I only see it after I push, so annoying.
> 
> Sorry about that. I use GitHub desktop and it can show that type of change, but it's very subtle and hard to notice.
> 
> I should make a separate PR to remove all the BOMs anyway. It has caused issues in the past.

It looks that I'm mistaken. The repo doesn't have BOMs, VS defaults to adding BOMs

**Comment by 1xFF** [comment]
> I forgot that the fixtures require these items to exist in the database. Which one of these is the correct one?
> 
> https://github.com/LizardByte/ThemerrDB/issues?q=is%3Aissue+is%3Aopen+steins%3Bgate+%282011%29

Well I'm biased so I'd say mine lol
It's the older one

**Comment by ReenigneArcher** [comment]
Thank you for being the first contributor!

**Comment by 1xFF** [comment]
My pleasure. The issue page is empty, but I look around to see if there's anything else to work on.

**Comment by ReenigneArcher** [comment]
Off the top of my head, these are some of the things that could be worked on:

- improve tests/increase code coverage
- replace depreciated ApiClient (in config html)
- prep for Jellyfin 10.9

**Pull request merged**